state-chain/pallets/cf-lp/src/migrations/v2.rs

```diff
@@ -0,0 +1,117 @@
+use frame_support::traits::OnRuntimeUpgrade;
+use sp_std::marker::PhantomData;
+
+#[cfg(feature = "try-runtime")]
+use frame_support::pallet_prelude::DispatchError;
+use sp_std::vec::Vec;
+
+use crate::{Config, HistoricalEarnedFees, Pallet};
+
+/// Migrates HistoricalEarnedFees to a DoubleMap.
+pub struct Migration<T>(PhantomData<T>);
+
+mod old {
+	use super::*;
+	use cf_chains::assets;
+	use cf_primitives::AssetAmount;
+	use codec::{Decode, Encode};
+	use frame_support::{pallet_prelude::ValueQuery, Twox64Concat};
+	use sp_std::vec;
+
+	#[derive(Encode, Decode, Default, Clone, PartialEq)]
+	struct EthMap {
+		eth: AssetAmount,
+		flip: AssetAmount,
+		usdc: AssetAmount,
+		usdt: AssetAmount,
+	}
+
+	#[derive(Encode, Decode, Default, Clone, PartialEq)]
+	struct DotMap {
+		dot: AssetAmount,
+	}
+
+	#[derive(Encode, Decode, Default, Clone, PartialEq)]
+	struct BtcMap {
+		btc: AssetAmount,
+	}
+
+	#[derive(Encode, Decode, Default, Clone, PartialEq)]
+	pub struct AssetMap {
+		eth: EthMap,
+		dot: DotMap,
+		btc: BtcMap,
```

Comment:
the order here in 1.3 is: eth, btc, dot

Reply:
Hmmm sure? On Berghain it's eth, dot, btc...

Reply:
I guess we're inconsistent elsewhere then...